src/compiler/utilities.ts

```diff
@@ -5606,8 +5606,7 @@ namespace ts {
             || kind === SyntaxKind.GetAccessor
             || kind === SyntaxKind.SetAccessor
             || kind === SyntaxKind.IndexSignature
-            || kind === SyntaxKind.SemicolonClassElement
-            || kind === SyntaxKind.MissingDeclaration;

```

Comment:
the parser can still add it in a class element context.. we have not changed the parser not to do that.. e.g. `;` in a class body would be parsed as MissingDeclaration.

Reply:
Looks like that is a `SemicolonClassElement`:
```ts
import ts = require("typescript");
const sourceFile = ts.createSourceFile("foo.ts", "class C { ; }", ts.ScriptTarget.ESNext);
const member = (sourceFile.statements[0] as ts.ClassDeclaration).members[0];
console.log(ts.SyntaxKind[member.kind]);;
```

Reply:
u are right.. sorry about that.